tests/settings.py

```diff
@@ -15,6 +15,8 @@
         'NAME': os.environ.get('DATABASE_NAME', 'oscar'),
         'USER': os.environ.get('DATABASE_USER', None),
         'PASSWORD': os.environ.get('DATABASE_PASSWORD', None),
+        'HOST': os.environ.get('DATABASE_HOST', 'localhost'),
```

Comment:
This causes problems if peer authentication is used instead of password authentication. Could you please leave out this change from this PR, and we can look separately at how to make this work with different auth mechanisms?